cmake/tracy.cmake

```diff
@@ -1,48 +1,34 @@
-
 # Built as outlined in Tracy documentation (pg.12)
 set(TRACY_HOME ${PROJECT_SOURCE_DIR}/tt_metal/third_party/tracy)
 
 add_subdirectory(${TRACY_HOME})
-set_target_properties(TracyClient PROPERTIES EXCLUDE_FROM_ALL TRUE)
+# set_target_properties(TracyClient PROPERTIES EXCLUDE_FROM_ALL TRUE)
```

Comment:
I'm pretty sure we can just remove this line. Might need to double check by just running a normal build and making sure it doesn't bring in Tracy, but it should be fine.